packages/gitgraph-js/src/gitgraph.ts

```diff
@@ -76,94 +77,99 @@ function createGitgraph(
   // including the message offset. Allows custom, flexible message height.
   // E.g. {20: 30} means for commit: y=20 -> y=30
   // Offset should be computed when graph is rendered (componentDidUpdate).
```

Comment:
ugh, I see I accidentally moved the comment - it should be above `commitYWithOffsets` - oops! I'll fix.